packages/wallet/test/services/ProviderTracker/TipTracker.test.ts

```diff
@@ -95,6 +100,43 @@ describe('TipTracker', () => {
     });
   });
 
+  it('syncStatus updates emitted only once connection is back up', () => {
```

Comment:
Please take a closer look here. I need another pair of eyes to validate the correctness of the test. Thanks!

Reply:
This test looks good to me